tests/rust/pragma_once.header.toml

```diff
@@ -0,0 +1,2 @@
+pragma_once = true
+no_includes = true
```

Comment:
Why is `no_includes` needed? It'd be nice to test the relative ordering of `#pragma once` and `#include`s.

Reply:
I did this following the include_guard option.
Removed `no_includes`